GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -134,7 +136,7 @@ import GTMSessionFetcher
   ///   not found, `Error.failedToRemoveAuthorizationFromKeychain` if some other error occurred.
   /// - Note: See `GTMKeychain.swift` for ``GTMAppAuthSwift/GTMKeychainError``s indirectly thrown by
   ///   calls to this method.
-  @available(macOS 10.15, *)
+  @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
Same as above.

Reply:
Updated to using `if #available` within the implementation.